passes/cmds/rename.cc

```diff
@@ -127,10 +130,12 @@ struct RenamePass : public Pass {
 		log("cells with private names.\n");
 		log("\n");
 		log("\n");
-		log("    rename -wire [selection]\n");
+		log("    rename -wire [selection] [-suffix <suffix>]\n");
 		log("\n");
 		log("Assign auto-generated names based on the wires they drive to all selected\n");
 		log("cells with private names. Ignores cells driving privatly named wires.\n");
+		log("The -suffix option can be used to set a suffix to the renamed cells.\n");
+		log("The default suffix is the cell type.\n");
```

Comment:
This message is now a little confusing. There is no mention of suffixes in the first two lines so the last line reads like the default is about what happens if you provide the -suffix option with no argument. How about something like the following message instead:

Assign auto-generated names based on the wires they drive to all selected cells with private names. Ignores cells driving privatly named wires. By default, the cell is named after the wire with the cell type as suffix.
The -suffix option can be used to set the suffix to the given string instead.

Reply:
I liked your text better so updated it.

Reply:
Thanks!